### PR TITLE
add hash-lock example

### PR DESCRIPTION
**Pull request by RetricSu**

### CODE REVIEW

File: website/docs/dapp/simple-lock.mdx [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+import TabItem from "@theme/TabItem";
+import TutorialHeader from "@components/TutorialHeader";
+import { LOCKOVERVIEW } from "./TutorialOverview.tsx";
+import SwitchToTestnet from "./_SwitchToTestnet.mdx";
+import StartDevnet from "./_StartDevnet.mdx";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import ImgContainer from "@components/ImgContainer";
+import CodeTabs from "./_CodeTabs.mdx";
+
+# Simple Lock Example
```
Comment (yfeng2824) [inline website/docs/dapp/simple-lock.mdx:16]: Can we rename this to "Build a Lock" to make it more consistent with other titles?

File: website/docs/dapp/TutorialOverview.tsx [hunk 2]
```diff
@@ -160,4 +160,53 @@ const DOBOVERVIEW: TutorialHeaderProps = {
   ],
 };
 
-export { TRANSFEROVERVIEW, WRITEOVERVIEW, TOKENOVERVIEW, DOBOVERVIEW };
+const LOCKOVERVIEW: TutorialHeaderProps = {
+  time: "10 - 15 min",
+  topics: [
+    {
+      label: "Full-Stack",
+      link: "/docs/tech-explanation/glossary#digital-object-dob",
```
Comment (yfeng2824) [inline website/docs/dapp/TutorialOverview.tsx:168]: Should we remove the link for this? (maybe need to make the link as an optional parameter)

File: website/docs/dapp/simple-lock.mdx (hunk 1 above)
Comment (RetricSu) [reply website/docs/dapp/simple-lock.mdx:16]: How about `Build a Simple Lock`?

File: website/docs/dapp/TutorialOverview.tsx (hunk 2 above)
Comment (RetricSu) [reply website/docs/dapp/TutorialOverview.tsx:168]: > Should we remove the link for this? (maybe need to make the link as an optional parameter)

yea i forget to remove this one

**Pull request merged**